src/core/fonts.js

```diff
@@ -827,7 +829,7 @@ var Font = (function FontClosure() {
       // with firefox and thuluthfont).
       if ((usedFontCharCodes[fontCharCode] !== undefined ||
            isProblematicUnicodeLocation(fontCharCode) ||
-           (isSymbolic && isIdentityUnicode)) &&
+           (isSymbolic && (isIdentityUnicode || !hasUnicodeValue))) &&
```

Comment:
Nit: Wouldn't `(isSymbolic && !hasUnicodeValue)` be a sufficient condition here, given that initially `hasUnicodeValue = false` and that it can only become `true` in the `!isIdentityUnicode` case?